output

```diff
@@ -0,0 +1,21 @@
+TERM_PROGRAM=Apple_Terminal
```

Comment:
You probably didn't mean to check this file in.  Would be good to add it to .gitignore if you expect it to be produced on each run.